### PR TITLE
Dashboard sidebar

### LINKED ISSUE #127: write dashboard sidebar (format + intro)
## Task Description

- write company/project intro and other info of interest into the sidebar (e.g. data source, linkedin linktree)

## User Stories

- As a user, I want to be able to access information about the dashboard and the creators behind it, to understand how to use the dashboard effectively and understand the dashboard within context.

## Relevant Files [If Available] 

-

### PR DESCRIPTION
**Pull request by ayeshaa63**
This PR creates a new file to organise the layout of the dashboard including a sidebar.

Closes #127

### CODE REVIEW

File: dashboard/layout.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+'''This file contains code to organise the layout of the dashboard e.g. the sidebar.'''
+
+import streamlit as st
+
+
+def set_page_config():
+
+    st.set_page_config(page_title="Arelm Court Dashboard", page_icon="🏛", layout="wide",
```
Comment (ErvinRex) [inline dashboard/layout.py:8]: Nice use of page configuration to improve the display of the dashboard.

File: dashboard/layout.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        st.write(
+            """This dashboard aims to summarise key insights from the court data we have collected. 
+            You can find out more about each case and judge.""")
+        st.subheader("Sources:")
+        st.link_button(
+            "All case law files", "https://caselaw.nationalarchives.gov.uk/")
+        st.link_button("All judges in the UK",
+                       "https://www.judiciary.uk/about-the-judiciary/who-are-the-judiciary/list-of-members-of-the-judiciary/")
+        st.subheader("Meet the team")
+        st.link_button("Linktree", "https: // linktr.ee/C10_Arelm")
```
Comment (ErvinRex) [inline dashboard/layout.py:28]: Lovely use of our linktree link!

Comment (RachelPun) [inline dashboard/layout.py:28]: I think we need to remove the extra spaces for the link to work

**Review by ErvinRex — APPROVED** [review]
Nice and concise sidebar, straight to the point and visually appealing!

**Review by RachelPun — CHANGES_REQUESTED** [review]
overall great! i think it's just the link?

File: dashboard/layout.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+    st.title("Courts Dashboard")
+
+
+def get_sidebar():
+
+    with st.sidebar:
+        st.title("Arelm")
+        st.subheader("Court Data Dashboard")
+        st.write(
```
[the remark this answers is not in the document]
Comment (ayeshaa63) [reply dashboard/layout.py:19]: I will investigate!

File: dashboard/layout.py (hunk 2 above)
Comment (ayeshaa63) [reply dashboard/layout.py:28]: Yes you're right I'll get rid of them

**Pull request merged**